packages/discovery-react-components/src/components/SearchResults/components/ResultElement/ResultElement.tsx

```diff
@@ -78,7 +78,7 @@ export const ResultElement: React.FunctionComponent<ResultElementProps> = ({
     className: elementBodyClassNames.join(' '),
     'data-testid': `search-result-element-body-${elementType}`,
     'aria-label': elementLabel,
-    ...(elementType === 'table' ? { role: 'img' } : {})
+    ...(elementType === 'table' ? { role: 'article' } : {})
```

Comment:
Please add a comment as to why this is here, since it's not readily evident from the code itself.